fxci/views/task_run_costs.view.lkml

```diff
@@ -0,0 +1,43 @@
+view: task_run_costs {
+  dimension: key {
+    primary_key:  yes
+    sql: CONCAT(${TABLE}.task_id, '-', ${TABLE}.run_id) ;;  }
```

Comment:
I created this view rather than using the one in the hub project, because I was getting errors about needing a `primary_key` when running queries.

Is there a way to set a primary key like this in `custom-namespaces.yml` so I can just use the generated view directly?

Reply:
Not something that is supported at the moment, but I added https://github.com/mozilla/lookml-generator/issues/1075 to keep track of this. Shouldn't be too hard to get added, and it sounds generally useful.